api/src/controllers/schema.ts

```diff
@@ -100,13 +97,24 @@ const schemaMultipartHandler: RequestHandler = (req, res, next) => {
 	req.pipe(busboy);
 };
 
+router.post(
+	'/apply',
+	asyncHandler(schemaMultipartHandler),
+	asyncHandler(async (req, _res, next) => {
+		const service = new SchemaService({ accountability: req.accountability });
+		const diff: SnapshotDiffWithHash = _res.locals['upload'];
```

Comment:
```suggestion
	asyncHandler(async (req, res, next) => {
		const service = new SchemaService({ accountability: req.accountability });
		const diff: SnapshotDiffWithHash = res.locals['upload'];
```

tiny tweak to remove the leading underscore since it was used to indicate `res` as an unused variable previously. 

Though understandably I do recall we do have rare usages of it to indicate private variable such as:

https://github.com/directus/directus/blob/381f1c9d8e6c1c799a19f34e12cbde88d525324a/app/src/composables/use-relation-multiple.ts#L45

Reply:
> tiny tweak to remove the leading underscore since it was used to indicate res as an unused variable previously.

Very sure the underscore was there for this reason 👍 😄